automatic/composer/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,30 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+
+$fileName = 'Composer-Setup.5.1.0.exe'
+
+$packageArgs = @{
+  packageName  = 'composer'
+  fileType     = 'exe'
+  file         = Get-Item $toolsPath\$fileName
+  checksum     = '80422FEF0EA310C0F565D6028F34010EEEBC96A4EDEC3175156D85F4F0EEEF10'
+  checksumType = 'sha256'
+  silentArgs   = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART'
+  softwareName = 'composer*'
+}
+
+try {
+  Install-ChocolateyInstallPackage @packageArgs
+  Get-ChildItem $toolsPath\*.exe | ForEach-Object { Remove-Item $_ -ea 0; if (Test-Path $_) { Set-Content "$_.ignore" '' } }
+}
+catch {
+
+  if ($env:ChocolateyExitCode -eq '1') {
```

Comment:
Is it guaranteed that the exit code will be `1` only when the php settings are incorrect?

Not saying that you need to change anything here, but just want to know.

Reply:
Yes, the exit code is always 1 when the installer aborts due to incorrect settings.